docs/en/notes/changelog.md

```diff
@@ -1 +1,77 @@
-# Changelog v3.x
+# Changelog of v3.x
+
+## v3.0.0rc0 (31/8/2022)
+
+We are excited to announce the release of MMDetection 3.0.0rc0.
+MMDet 3.0.0rc0 is the first version of MMDetection 3.x, a part of the OpenMMLab 2.x projects.
+Built upon the new [training engine](https://github.com/open-mmlab/mmengine),
+MMDet 3.x unifies the interfaces of dataset, models, evaluation, and visualization with faster training and testing speed.
+It also provide a general semi-supervised object detection framework, and more strong baselines.
+
+### Highlights
+
+1. **New engines**. MMDet 3.x is based on [MMEngine](https://github.com/open-mmlab/mmengine), which provides a general and powerful runner that allows more flexible customizations and significantly simplifies the entrypoints of high-level interfaces.
+
+2. **Unified interfaces**. As a part of the OpenMMLab 2.x projects, MMDet 3.x unifies and refactors the interfaces and internal logics of train, testing, datasets, models, evaluation, and visualization. All the OpenMMLab 2.x projects share the same design in those interfaces and logics to allow the emergence of multi-task/modality algorithms.
+
+3. **Faster speed**. We optimize the training and inference speed for common models, achieving up to xx% speedup for training and xx% for inference. Please refer to [benchmark](./benchmark.md#comparison-with-detectron2) for details.
```

Comment:
`xx% speedup` Do we need to write specific metrics now?

Reply:
It would be better. I will clean it for now.